docs/knative-eventing-mesh/08-01-configure-kafka-channel.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Configure Kafka Channel
+type: Tutorials
+---
+
+Instead of the default Channel implementation, you can use the Knative-compatible [Kafka Channel](https://github.com/kyma-incubator/knative-kafka). To ensure Kafka works properly, you must:
+
+* Install the `knative-eventing-kafka`  component to deploy the Kafka controller
+* Set up a Kafka cluster using Azure Event Hubs
+* Create a Secret whic the controller uses to communicate with the cluster
+
+**NOTE**: Although Event Hubs and Kafka are nearly identical as concepts, they use different naming convention. To avoid confusion, read [this document](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-for-kafka-ecosystem-overview#kafka-and-event-hub-conceptual-mapping).
+
+## Prerequisites
+
+Before Kyma installation, [enable](/root/kyma/#configuration-custom-component-installation) the `knative-eventing-kafka` custom component.
+
+## Steps
+
+Follow these steps:
+
+1. Use the Azure portal to create a [resource group](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-a-resource-group) where your will place your cluster.
+
+2. Create an [Event Hub namespace](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create#create-an-event-hubs-namespace) which is an Event Hub representation of the cluster.
+
+  **NOTE**: You can also use other providers, such as [Confluent Cloud](https://www.confluent.io/confluent-cloud) or install Kafka [locally](https://kafka.apache.org/quickstart), but bear in mind that these are experimental options.
+
+3. Export the variables. To retrieve the credentials, go to Azure Portal > **All services** > **Event Hubs** and select your Event Hub. 
+
+  ```bash
+  $ export kafkaBrokers={BROKER_URL}
+  $ export kafkaNamespace={KAFKA_CLUSTER_NAME}
+  $ export kafkaPassword={PASSWORD}
+  $ export kafkaUsername={CONNECTION_STRING}
+  $ export kafkaProvider=azure
+  ```
+2. Apply the override which creates the Kafka Secret. 
+
+  ```bash
+  $ cat << EOF | kubectl apply -f -
```

Comment:
This is tricky. It only works then Kyma installation has already started. Because the namespace `kyma-installer` does not exist before, kubectl would give an error then running the command to early.

We recently figured out, that Kyma CLI supports overrides. 

```bash
$ kyma install -o <override.yaml> <- this file would have the installation-overrides
```

It is already used in[ test-infra](https://github.com/kyma-project/test-infra/blob/798edeb550b640cb74d1621f49c009d319bc112a/prow/scripts/cluster-integration/kyma-integration-gardener-azure.sh#L171) for the `kyma-integration-gardener-azure` job.